### ISSUE TITLE
LicenseCompareHelper: CC-BY-4.0 license text published by Creative Commons not recognised

### ISSUE THREAD

**Issue by pmonks** [spdx/license-list-XML#1960]
The [license text for `CC-BY-4.0` published by Creative Commons](https://creativecommons.org/licenses/by/4.0/legalcode.txt) is not being recognised by `org.spdx.utility.compare.LicenseCompareHelper.isTextStandardLicense()`.  Specifically, the "difference message" in the returned  `DifferenceDescription` object says:

```
Normal text of license does not match starting at line spdx/Spdx-Java-Library#5 column spdx/Spdx-Java-Library#9 "commons" when comparing to template text "

By exercising the Licensed Rights (defined below), You accept and agree to be bound by the terms a...".  Last optional text was not found due to the optional difference:
	Normal text of license does not match starting at line spdx/Spdx-Java-Library#5 column spdx/Spdx-Java-Library#29 "(" when comparing to template text " Creative Commons Attribution 4.0 International Public License

".  Last optional text was not found due to the optional difference:
	Normal text of license does not match starting at line spdx/Spdx-Java-Library#34 column spdx/Spdx-Java-Library#5 "licensed" when comparing to template text " Creative Commons Corporation ("Creative Commons") is not a law firm and does not provide legal serv...".  Last optional text was not found due to the optional difference:
	Normal text of license does not match starting at line spdx/Spdx-Java-Library#34 column spdx/Spdx-Java-Library#5 "licensed" when comparing to template text " Creative Commons Corporation ("Creative Commons") is not a law firm and does not provide legal serv...".  Last optional text was not found due to the optional difference:
	Normal text of license does not match starting at line spdx/Spdx-Java-Library#1 column spdx/Spdx-Java-Library#12 "4" when comparing to template text "Creative Commons"
```

Note: tested with Spdx-Java-Library v1.1.5.

**Comment by pmonks** [comment]
Just a head's up that I'm seeing similar issues matching these `CC-BY-*-4.0` texts too, and assume they're caused by the same underlying issue in the matching templates:

- [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/legalcode.txt)
- [CC-BY-NC-4.0](https://creativecommons.org/licenses/by-nc/4.0/legalcode.txt)
- [CC-BY-NC-SA-4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode.txt)
- [CC-BY-ND-4.0](https://creativecommons.org/licenses/by-nd/4.0/legalcode.txt)
- [CC-BY-NC-ND-4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.txt)

**Comment by swinslow** [comment]
Fixed in #2280
